themes/stanford_basic/templates/block/block--system-menu-block.html.twig

```diff
@@ -45,8 +45,9 @@
   {% if not configuration.label_display %}
     {% set title_attributes = title_attributes.addClass('visually-hidden') %}
   {% endif %}
+
   {{ title_prefix }}
-  <h2{{ title_attributes.setAttribute('id', heading_id) }}>{{ configuration.label }}</h2>
+  <h2{{ title_attributes.setAttribute('id', heading_id) }} {% if not configuration.label_display %}aria-hidden=true{% endif %}>{{ configuration.label }}</h2>
```

Comment:
I'd like to put this in a preprocess hook that applies to all blocks on every page, not just this one.

Reply:
Ok.  How about this @pookmish?